Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.13 as modules
+ADD ./go.mod /m/
+RUN cd /m && go mod download
+FROM golang:1.13 as builder
+
+RUN mkdir -p /opt/resource/
+
+COPY --from=modules /go/pkg/ /go/pkg/
+
+WORKDIR /opt/resource/
+COPY cmd             cmd
+COPY config          config
+COPY middleware      middleware
+COPY model           model
+COPY server          server
+COPY storage         storage
+COPY go.*            ./
+
+WORKDIR /opt/resource/cmd/umserver
+RUN go build -o /opt/services/example-app .
+
+FROM alpine:3.7
+COPY --from=builder /opt/services/example-app /opt/services/example-app
+CMD /opt/services/example-app
```

Comment:
example-app is not good name for your service

Reply:
rename service